.travis.yml

```diff
@@ -38,6 +38,7 @@ script:
   - ./container_build.py kmod_buildtest
   - cd core && ./all_test && cd ..
   - ./sanity_check.sh
+  - cd bessctl && ./bessctl -- daemon start -- run testing/run_module_tests
```

Comment:
Maybe change this to:

bessctl/bessctl -- daemon start -- run testing/run_module_tests

Unless there's some reason that won't work.